### PR TITLE
Colab test 01

### PR DESCRIPTION
**Pull request by lamehandle**
Moving forward I will just push to current PR.

### CODE REVIEW

**Review by p-p-m — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/env/
+/.ipynb_checkpoints/
```
Comment (p-p-m) [inline .gitignore]: [critical] 

1. Looks like `/.ipynb_checkpoints/` does not work. I see this changes in git diff.
2. Please add `.idea/` to gitignore too.

File: candle_stick_chart.ipynb [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
...cOHNCJEyd05MgRrV69Wtddd51rucOHD+vTTz/VDTfcwC0o/kDtzKFu5lE7c6ibedQO1yTv3oYThmEYR44cMVq3bm1MmzbNNW379u1G9+7djUaNGhXoZtglFbUzh7qZR+3MoW7mUTtcq7jwxwtOnjwp6c/bS5w8eVKpqalq0KCBa5mYmBg999xzCg0N1dKlSyVxArdE7cyibuZRO3Oom3nUDiUFIdNi6enpatiwof773/+6bi8RERGh6667Tps3b3YtZ7fbFRcXJ19fX+3atUuSSvx5NdTOHOpmHrUzh7qZR+1QkhAyLebj46OEhAQ9+uijWrVqlWtakyZNtHHjRtdtJ6S8PxiVKlVSaGioDMMo8f+lUjtzqJt51M4c6mYetUNJwoU/XpCenq5XX31Vb7/9tv71r3+pa9euOnbsmB5++GFlZmaqSZMmat68ub744gstWrRIn376qerUqVPU3S4WqJ051M08amcOdTOP2qHEKNxTQK9NWVlZRkZGhtu0Q4cOGSNGjDDCwsKMDz/80DAMwzh69Kjx97//3ejYsaPRpEkTIyEhwfjhhx+KosvFBrUzh7qZR+3MoW7mUTuUVITMK5SSkmK0bt3a+L//+z9j4cKFxrp161zzzpw5YwwfPtwICwsz/vvf/xqGYRg5OTlGbm6ukZ6ebmRlZRVRr4sHamcOdTOP2plD3cyjdijJfC8/1omLcTqdWrBggZKTkxUYGKjXX39dp0+fVlhYmG644Qb16dNH9957r8qVK6cHHnhAZcqUUfv27SVJFSpUKOLeFy1qZw51M4/amUPdzKN2KOk4J/MKpaamatq0adq7d69q1qypAQMGaPHixfryyy+1fft2hYWFqVq1avr++++Vnp6u5cuXq1WrVkXd7WKB2plD3cyjduZQN/OoHUoyRjKvUGRkpAYPHqypU6fqf//7n2rWrKmnn35akrR161YdPnxY8+fPV4UKFZSenq7y5csXcY+LD2pnDnUzj9qZQ93Mo3YoyRjJtMjhw4c1depUffPNN+rSpYuGDRvmmpeTkyOn06nMzEyFh4cXYS+LJ2pnDnUzj9qZQ93Mo3YoiQiZFkpNTdWrr76q7777Tl26dNHQoUMlSbm5ufL1ZdD4UqidOdTNPGpnDnUzj9qhpCFkWiz/j8gPP/ygW265Rc8880xRd+mqQe3MoW7mUTtzqJt51A4lCZ
```
Comment (p-p-m) [inline candle_stick_chart.ipynb:85]: I am not sure that we want to keep this in git. What does this file correspond to?

**Review by p-p-m — CHANGES_REQUESTED** [review]

File: candle_stick_chart.ipynb (hunk 2 above)
Comment (lamehandle) [reply candle_stick_chart.ipynb:85]: This is a notebook with example candle stick charts in it. Basically it's a test.

File: .gitignore (hunk 1 above)
Comment (lamehandle) [reply .gitignore]: /.ipynb_checkpoints/ was added to my gitignore in error

**Review by p-p-m — APPROVED** [review]

**Pull request merged**